tests/runtime/probe

```diff
@@ -119,6 +113,18 @@ EXPECT progs: 1
 TIMEOUT 5
 REQUIRES /usr/sbin/bpftool
 
+NAME kprobe_module
+PROG kprobe:vmlinux:vfs_read { printf("SUCCESS %d\n", pid); exit(); }
+EXPECT SUCCESS [0-9][0-9]*
+TIMEOUT 5
+AFTER ./testprogs/syscall read
+
+NAME kprobe_module_wildcard
+PROG kprobe:vmlinu*:vfs_read { printf("SUCCESS %d\n", pid); exit(); }
+EXPECT SUCCESS [0-9][0-9]*
+TIMEOUT 5
+AFTER ./testprogs/syscall read
```

Comment:
Is it worth adding a runtime test for a kernel module that's either not loaded and/or doesn't exist?

Reply:
Good idea; we need to make sure that not only does the PR *work* the same way as without a module being specified, but that it also *fails* in the same way.

Reply:
...Adding this revealed a bit of a problem. If I specify a kernel module, attaching appears to "succeed." That is, the `bpf_attach_kprobe` call (passing in `module:function` as an argument) succeeds in creating a perf fd. I'll continue discussing on the affected line.